### ISSUE TITLE
Touch and move not working as expected in Android browsers

### ISSUE THREAD

**Issue by drenther** [edbentley/replay#18]
## Issue Description
Touch and move should result in (If I am not wrong at understanding the API :P) webinputs.pressed staying true with updates to x and y value based on the movements. It doesn't work as expected in Android browsers.

May need some investigation around why. I will try and update as much as I find.

## Steps to reproduce
Codesandbox that show a small case of what I am talking about - https://codesandbox.io/s/replay-touchmove-example-r9om6

Desktop (Works) - Try pressing and moving the mouse around in desktop and it works as expected
iOS (Chrome) - Tap and move around. Works as expected
Android (Chrome) - Tap and move around. It doesn't respond after first 200-300 ms.
Android (Firefox) - Doesn't register the tap at all

## Might be related
May have something to do with this - https://support.google.com/chrome/thread/7838470?hl=en

**Comment by edbentley** [comment]
Interesting, thanks for reporting! I don't have an Android device so never tested it 😅 

I couldn't get your Codesandbox to work, but I tested the standard [replay-starter-ts](https://codesandbox.io/s/github/edbentley/replay-starter-ts) Codesandbox on an Android emulator.

It worked fine in the emulator for Android Chrome, but I reproduced Android Firefox not registering touches.

replay-web uses [pointer-down](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/pointerdown_event) to register touches, and for Android Firefox it states:

> this feature is behind the dom.w3c_pointer_events.enabled preference (needs to be set to true). To change preferences in Firefox, visit about:config.

I toggled that preference and it did indeed start working. Strangely that says Safari doesn't support it, but it works on Safari. Regardless, perhaps we need to use an alternative to pointer-down.
